### ISSUE TITLE
Make sure Discord/Adobe app is running...

### ISSUE THREAD

**Issue by Changshire** [smokes/adobe-rpc#6]
Both Adobe premiere Pro and Discord are running, but the software isn't having any of it, i used the exe one.

**Comment by smokes** [comment]
The exe isn't up to date with the github repository, I may release a newer one if I ever have time. For now, you can use the python version. Sorry.

**Comment by carburator03** [comment]
Same with Adobe XD

**Comment by smokes** [comment]
@hydraPNG I don't think we will support Adobe Xd anytime soon because they do not display information that we need for the RPC in the title bar. If Adobe ever decides to fix this, we will easily include it

https://adobexd.uservoice.com/forums/353007-adobe-xd-feature-requests/suggestions/33060586-show-project-title-file-name-for-each-window-whe

**Comment by carburator03** [comment]
@smokes OK, thanks for the reply 😃
